tasks/__init__.py

```diff
@@ -556,6 +556,17 @@ def test_addons(ctx):
     for addon in settings.ADDONS_REQUESTED:
         module = os.path.join(settings.BASE_PATH, 'addons', addon)
         modules.append(module)
+
+    print(modules)
+    test_module(ctx, module=modules)
+
+@task
+def test_evernote(ctx):
```

Comment:
I created `test_evernote` to make it easy for me to run only evernote tests.  I didn't see an easy way to do so with the existing tests.